packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -491,12 +498,16 @@ export class FeatureDevController {
             }
         }
     }
-    private workOnNewTask(
+    private async workOnNewTask(
         message: any,
         remainingIterations: number = 0,
         totalIterations?: number,
         isStoppedGeneration: boolean = false
     ) {
+        const hasDevFile = await checkForDevFile(
+            (await this.sessionStorage.getSession(message.tabID)).config.workspaceRoots[0]
```

Comment:
Is there some UI indication that only the first workspace root is used?

Reply:
no, good point, let me think about how to handle this

Reply:
We will follow up with this, most likely going to modify from strings